src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java

```diff
@@ -76,8 +76,8 @@ private DefaultJudgmentSDOOrderForm getDefaultJudgmentFormHearing(CaseData caseD
             .caseNumber(caseData.getLegacyCaseReference())
             .disposalHearingBundleDJ(caseData.getDisposalHearingBundleDJ())
             .disposalHearingBundleDJAddSection(nonNull(caseData.getDisposalHearingBundleDJ()))
-            .typeBundleInfo(nonNull(caseData.getDisposalHearingBundleDJ())
-                                ? fillTypeBundleInfo(caseData.getDisposalHearingBundleDJ().getType()) : null)
+            //.typeBundleInfo(nonNull(caseData.getDisposalHearingBundleDJ())
```

Comment:
As the other commented out lines, either remove them or add a comment so that other devs know why they should be there